lib/statsd/instrument/prometheus/aggregators/last_value.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module StatsD
+  module Instrument
+    module Prometheus
+      module Aggregators
+        class LastValue < Base
+          def aggregate
+            datagrams.last
```

Comment:
For gauges: https://github.com/coralogix/statsd-coralogix-backend/blob/master/lib/coralogix.js#L181